pkg/cli/rack.go

```diff
@@ -426,6 +433,47 @@ func RackUpdate(rack sdk.Interface, c *stdcli.Context) error {
 	return c.OK()
 }
 
+func RackSync(rack sdk.Interface, c *stdcli.Context) error {
+	c.Startf("Synchronizing rack API URL...")
+	c.Writef("\n")
+
+	host, err := currentHost(c)
+	if err != nil {
+		c.Fail(err)
+	}
+	rname := currentRack(c, host)
+
+	if c.String("name") != "" {
```

Comment:
for non-managed console racks. Users can pass the rack name and it will fetch from CF directly.